citizenvoice/survey_design/models/__init__.py

```diff
@@ -1,5 +0,0 @@
-from .answer import Answer
```

Comment:
Did you delete this file? Keep it even if it is empty

Reply:
Yes, these models are not/ should not be used anymore, as the API integration replaces this. Should I rather not delete unused files/ files that should no longer be used in the future?

Reply:
Please, keep it to maintain the Django structure of an app